mage_ai/frontend/components/PipelineDetail/UploadPipeline/index.style.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
```

Comment:
@PopaRares Instead of creating a duplicate file with these components, can you reuse the components from `mage_ai/frontend/components/FileBrowser/UploadFiles/index.style.tsx`?